homework_1/task_02/taxes/taxes_test.go

```diff
@@ -0,0 +1,25 @@
+package taxes_test
+
+import (
+	"code-cadets-2021/homework_1/task_02/taxes"
+	"fmt"
+	"testing"
+
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+func TestCalculateTax(t *testing.T) {
+	for idx, tc := range getTestCases() {
+		Convey(fmt.Sprintf("Given test case #%v: %+v", idx, tc), t, func() {
+
+			output, err := taxes.CalculateTax(tc.input, tc.percentages, tc.amounts)
```

Comment:
I'm pretty sure this test doesn't work anymore (since the `CalculateTax` function definition has changed).